src/KnowbaseItem.php

```diff
@@ -1920,7 +1920,13 @@ public static function showList($options, $type = 'search')
             // Pager
             $parameters = "start=" . $params["start"] . "&amp;knowbaseitemcategories_id=" .
                         $params['knowbaseitemcategories_id'] . "&amp;contains=" .
-                        $params["contains"] . "&amp;is_faq=" . $params['faq'];
+                        $params["contains"];
+
+            if ($type != 'search' && $type != 'browse') {
+                $parameters .= "&amp;is_faq=" . $type;
```

Comment:
It seems strange to me, because `is_faq` expects a boolean whereas `$type` will have a string value (‘myunpublished’, ‘allunpublished’, ‘allmy’, ...).